queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
According to the requirement, you are to select name and not * from animals, kindly make the correction before submitting again 😄 

![image](https://github.com/mohashyne/vet_clinic/assets/20537712/0d03b393-298d-4663-8889-61b40c51d33d)